.gitignore

```diff
@@ -39,6 +39,9 @@ htmlcov
 .project
 .pydevproject
 
+# PyCharm
+.idea
```

Comment:
Please follow this guide rather than modifying the `.gitignore` of every project you contribute to: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Ah OK, sorry. I removed that line and solved it with a global gitgnore instead.